presto-spi/src/main/java/io/prestosql/spi/connector/ConnectorMetadata.java

```diff
@@ -57,12 +59,14 @@ default boolean schemaExists(ConnectorSession session, String schemaName)
     /**
      * Returns a table handle for the specified table name, or null if the connector does not contain the table.
      */
+    @Nullable
     ConnectorTableHandle getTableHandle(ConnectorSession session, SchemaTableName tableName);
 
     /**
      * Returns a table handle for the specified table name, or null if the connector does not contain the table.
      * The returned table handle can contain information in analyzeProperties.
      */
+    @Nullable
```

Comment:
if this is nullable, why default impl does not return null?

Reply:
Because it does not support this operation, while null return value is still a correct value to return.